cms/djangoapps/contentstore/signals/apps.py

```diff
@@ -0,0 +1,22 @@
+"""
```

Comment:
Comment from a passerby:
Note that the `apps.py` file should be in the top-level directory of `contentstore`, not within the `signals` specific subdirectory.   For reference, see django's documentation on [Applications](https://docs.djangoproject.com/en/1.11/ref/applications/).  `apps.py` is meant to contain all sorts of django-app specific stuff, not just related to signals.  Thanks!